apps/swirl-docs/src/components/Tags/index.tsx

```diff
@@ -4,6 +4,24 @@ import { FunctionComponent } from "react";
 export interface TagProps {
   content: string;
   scheme?: "critical" | "info" | "success" | "warning" | "default";
+  httpTag?: boolean;
+}
+
+export function mapHttpMethodToTagContent(
+  httpMethod: string
+): TagProps["content"] {
+  switch (httpMethod.toUpperCase()) {
+    case "GET":
+      return "GET";
+    case "POST":
+      return "POST";
+    case "PUT":
+      return "PUT";
+    case "DELETE":
+      return "DEL";
+    default:
+      return "GET";
```

Comment:
**issue:** Would return an incorrect label for e.g. methods "OPTIONS", "HEAD" or "TRACE".